.github/workflows/hatch_ci.yml

```diff
@@ -6,20 +6,13 @@ on:
       min-python-version:
         type: string
         required: false
-        default: "3.9"
+        default: "3.10"
         description: "Minimum Python version to test against."
       max-python-version:
         type: string
         required: false
         default: "3.12"
         description: "Maximum Python version to test against."
-    secrets:
```

Comment:
This looks to be unintentional. (deleting the secret references.) It's still being used in this workflow.

Reply:
Mistake. Added back the secrets